app/controllers/reorder_sub_sections_controller.rb

```diff
@@ -1,27 +1,37 @@
 class ReorderSubSectionsController < ApplicationController
   before_action :require_coronavirus_editor_permissions!
-  before_action :redirect_to_index_if_slug_unknown
   layout "admin_layout"
 
   def index
     coronavirus_page
   end
 
   def update
+    success = true
     current_positions
```

Comment:
Does this line do anything? looks superfluous from my quick glance (sorry can see it's not part of your diff)

Reply:
Yeah, it turns out `current_positions` isn't being used anywhere. I'll remove it.